### PR TITLE
added windows to INSTALL.md

### PR DESCRIPTION
**Pull request by dgquintas**

### CODE REVIEW

**Comment by dgquintas** [comment]
Should I also backport this to the v1.0.x branch?

**Comment by hsaliak** [comment]
yes, if possible.

**Comment by dgquintas** [comment]
Will do, but I need to get this one merged first.

**Comment by hsaliak** [comment]
LGTM

File: INSTALL.md [hunk 1]
```diff
@@ -55,3 +55,55 @@ gRPC C Core library.
  $ make
  $ [sudo] make install
 ```
+
+##Windows
+
+There are several way to build under Windows, of varying complexity depending on
```
Comment (ah-) [inline INSTALL.md]: ways

Comment (dgquintas) [inline INSTALL.md]: nice. done.

**Pull request merged**